src/loader.c

```diff
@@ -61,20 +82,66 @@ load_impl_library()
             fprintf(stderr, "wpe: could not load the impl library. Is there any backend installed?: %s\n", dlerror());
             abort();
         }
+        wpe_loader_set_library_name("libWPEBackend-default.so");
     }
 #endif
 
     s_impl_loader = dlsym(s_impl_library, "_wpe_loader_interface");
 }
 
+__attribute__((visibility("default")))
+void
+wpe_loader_init(const char* library_name)
```

Comment:
Could this be less fatal and just return true/false depending on the outcome? This might allow more dynamic search for any implementation library that could be used.

Reply:
I made it fatal because everything else in loader aborts on failure. But this is an API call so it probably makes sense to make it return false on failure instead.